lib/fastlane/plugin/revenuecat_internal/helper/versioning_helper.rb

```diff
@@ -297,7 +337,7 @@ def self.append_phc_version_if_necessary(append_phc_version_if_next_version_is_n
       private_class_method def self.get_type_of_change_from_pr_info(pr_info)
         pr_info["labels"]
           .map { |label_info| label_info["name"].downcase }
-          .select { |label| Helper::GitHubHelper::SUPPORTED_PR_LABELS.include?(label) }
+          .select { |label| Helper::GitHubHelper::SUPPORTED_PR_LABELS.include?(label) || label.start_with?("feat:") }
```

Comment:
If we do rename `features` to something else, we might also want to rename these labels